bite-size-notes/week-1-e-linearization.md

```diff
@@ -0,0 +1,250 @@
+# Linearization
+
+**Scope: Linearization in Multivariable Control System** 
+- Many control models in real world are mostly made up of multiple states and a inputs. Using simple examples, pendulum, which is often covered in classes, we will study how linearization can be applied in multivariable control system.
```

Comment:
Hi~ Thank you for your feedback. Is the reviewing note for four of us(people in week 1)?

Reply:
My understanding is we each review the "next" person in this week's list - so for you it would be to review week-1-a notes (I did yours since there is no week-1-d)

Reply:
Oh~ do you when is the due date for this review?

Reply:
It's in the course syllabus - exactly one week after 1st draft notes are due, iirc

Reply:
Oh~ I see. I found the due date but I can't not find that we should review the "next person", do you know where the information is? If it is not, may be we should review overall maybe?

Reply:
It's in the sign-up document for these notes